### DISCUSSION TITLE
Echovald Gold Farm

### DISCUSSION THREAD

**Discussion by Weru** [blish-hud/Blish-HUD#657] [Requested Marker Packs]
Could you create a pathing map for the route Unskippable Cutscene has for Echovald, 
GW2 | Echovald Wilds Gold Farm -  https://youtu.be/7SXbg_C-9Lk

**Comment by Metallis** [comment]
I'd think I could knock this out pretty quickly, just need to toy around with trails a bit more.

**Reply by Metallis** [reply]
First attempt at the trail done. Weru, can you give it a shot and let me know your thoughts.
[echo-resin.zip](https://github.com/blish-hud/Blish-HUD/files/8599077/echo-resin.zip)

**Comment by Metallis** [comment]
This request is now complete, per [this release](https://github.com/Metallis/Metal-Marker-Myriad/releases/tag/v1.1). Please download the latest version of the Metal Marker Myriad pack in the Blish HUD UI.
